streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java

```diff
@@ -116,6 +116,7 @@ public void shutdown() {
     }
 
     @Test
+    @SuppressWarnings("unchecked")
```

Comment:
Auto-configuration has nothing to do with state restoring and testing auto-configuration should be be included in this test at all.